### ISSUE TITLE
Implement Wayfinder 2.1

### ISSUE THREAD

**Issue by hobuobi** [Recidiviz/public-dashboard#523]
The Wayfinder is a feature that allows users to preview the content of a given narrative. 

The current Wayfinder has a few main issues: 
- It's very small, which presents a possible accessibility failure. 
- While it allows users to preview on hover, there is no view that allows a user to see all of the content at one go. 
- It does not exist on mobile, where users might benefit greatly from the persistent context. 

### Scope 

Wayfinder 2.1 should make it significantly easier for users to dive into and navigate the narratives, including: 
- P0: Expanding the presence of the Wayfinder to a persistent bar along the bottom of the page that highlights the "forward/backwards" buttons and the title of the current narrative. 
- P0: Implementing the Wayfinder on both desktop and mobile. 
- P0: Including a "Table of Contents" that allows the user to preview all of the sections of a narrative in one go. This should be activated by clicking/tapping a dedicated button. 
- P1: Implementing a first-run experience that introduces new users to the Wayfinder. 
- P1: Nudge users to proceed to the next section after a few seconds of landing on a narrative via a visual cue on the "next" button (pulsing, flashing, tooltip, etc). 

### Design
![Screen (27).png](https://images.zenhubusercontent.com/5efa0f1acabfc83e776b1102/cccb702b-40cd-4985-aad8-8095b42ee15b)
![iPhone 12 Mini.png](https://images.zenhubusercontent.com/5efa0f1acabfc83e776b1102/9efb8939-d570-45a7-ba04-e8f831447517)
